app/views/votes/_voting.html.erb

```diff
@@ -1,11 +1,10 @@
 <% if @user_votes.include? vote %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
     <%= link_to reason, quote_vote_path(quote, vote, { vote: { value: value } } ), method: :patch %>
   <% end %>
   <%= link_to "Erase my vote", quote_vote_path(quote, vote), method: :delete %>
 <% else %>
-  <% Vote::VOTE_TYPES.each do |reason, value| %>
+  <% t(:votes).each do |value, reason| %>
```

Comment:
Okay so I implemented this in a different way now..  The way Richard wanted me to.  Please check that out.  Probably better than what I did in #32 because while we do have to mess around in two files here it gives us more flexibility than the other implementation.